pmix/borrow.py

```diff
@@ -185,21 +186,38 @@ def borrow_cli():
     )
 
     args = parser.parse_args()
-    ignore = set(args.ignore) if args.ignore else None
-    add = sorted(list(set(args.add))) if args.add else None
-    correct = args.correct if args.correct else []
-    translation_dict = create_translation_dict(args.xlsxfile, correct)
-    if args.merge is None and args.merge_all is None:
-        write_translation_file(translation_dict, args.outpath, add,
-                               args.diverse)
+    borrow(merge=args.merge,
+           merge_all=args.merge_all,
+           correct=args.correct,
+           no_diverse=args.no_diverse,
+           diverse=args.diverse,
+           add=args.add,
+           ignore=args.ignore,
+           carry=args.carry,
+           outpath=args.outpath,
+           xlsxfiles=args.xlsxfiles)
+
+
+def borrow(merge=[], merge_all=[], correct=[], no_diverse=False,
+           diverse=None, add=[], ignore=[], carry=False, outpath='',
+           xlsxfiles=[]):
```

Comment:
Setting mutable default values for a routine is a no-no in Python.

Reply:
Yes, that's a good idea. Let's use the new strong typing. I've just been doing this instead for the time being. Maybe you can give me a brief crash course or link me to some documentation that you've looked at before to get me started.

Reply:
Done~